services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -3,39 +3,40 @@
 import shutil
 import time
 from pathlib import Path
-from typing import Dict
+from typing import Dict, Optional
 
 import aiodocker
 import aiopg
 import attr
-from celery.utils.log import get_task_logger
 from packaging import version
 from tenacity import retry, stop_after_attempt
 
+from celery.utils.log import get_task_logger
 from servicelib.utils import fire_and_forget_task, logged_gather
 from simcore_sdk import node_data, node_ports
 from simcore_sdk.node_ports.dbmanager import DBManager
 
 from . import config, exceptions
+from .boot_mode import get_boot_mode
 from .log_parser import LogType, monitor_logs_task
 from .rabbitmq import RabbitMQ
-from .boot_mode import get_boot_mode
+from .utils import get_volume_mount_point
 
 log = get_task_logger(__name__)
 
 
 @attr.s(auto_attribs=True)
 class TaskSharedVolumes:
-    input_folder: Path = None
-    output_folder: Path = None
-    log_folder: Path = None
+    input_folder: Optional[Path] = None
+    output_folder: Optional[Path] = None
+    log_folder: Optional[Path] = None
 
     @classmethod
     def from_task(cls, task: aiopg.sa.result.RowProxy):
         return cls(
-            Path.home() / f"input/{task.job_id}",
-            Path.home() / f"output/{task.job_id}",
-            Path.home() / f"log/{task.job_id}",
+            config.SIDECAR_INPUT_FOLDER / f"{task.job_id}",
```

Comment:
do you really need the f-string? ``task.job_id`` is not a string on its own?

Reply:
just why is this an issue? even if it was a string I mean

Reply:
not an issue , just a question ;-)